scripts/pre_commit_linter.py

```diff
@@ -515,7 +515,7 @@
 EXCLUDED_PATHS = (
     'third_party/*', 'build/*', '.git/*', '*.pyc', 'CHANGELOG',
     'integrations/*', 'integrations_dev/*', '*.svg', '*.gif',
-    '*.png', '*.zip', '*.ico', '*.jpg', '*.min.js',
+    '*.png', '*.zip', '*.ico', '*.jpg', '*.min.js', 'backend_prod_files/'
```

Comment:
Should we omit webpack_bundles/ too? Are there others?

Reply:
sure, added that!

Reply:
I don't think there others, though I can continue searching tmw.